tests/functional/test_batching.py

```diff
@@ -49,10 +48,10 @@ def test_run_inference(self, resnet_8_batch_model_downloader,
         print("Downloaded model files:", resnet_8_batch_model_downloader)
 
         # Starting docker with ie-serving
-        result = start_server_batch_model
-        print("docker starting status:", result)
-        time.sleep(15)  # Waiting for inference service to load models
-        assert result == 0, "docker container was not started successfully"
+        container = start_server_batch_model
+        running, logs = wait_endpoint_setup(container)
+        print("Logs from container: ", logs)

```

Comment:
maybe this print, could be inside - `wait_endpoint_setup(container)`?

Reply:
or maybe this wait, should be part of start_server?
Because I see too many Repeated lines here